bot/exts/filtering/_utils.py

```diff
@@ -176,10 +174,7 @@ def __init__(self):
     def __init_subclass__(cls, **kwargs):
         def inherited(attr: str) -> bool:
             """True if `attr` was inherited from a parent class."""
-            for parent in cls.__mro__[1:-1]:  # The first element is the class itself, last element is object.
-                if hasattr(parent, attr):  # The attribute was inherited.
-                    return True
-            return False
+            return any(hasattr(parent, attr) for parent in cls.__mro__[1:-1])
```

Comment:
This deletes comments

Reply:
Updated in https://github.com/python-discord/bot/pull/2523/commits/6347cf6c1337ab97224263c6891531ce97c0e169